### DISCUSSION TITLE
Zoom-Fenster verschieben

### DISCUSSION THREAD

**Discussion by henningTheDev** [SemTalkOnline/SemTalkOnline#9] [Q&A] [answered]
Wie kann ich das Zomm-Fenster verschieben, damit es nicht unter einem Dialog begraben wird?

**Comment by SemTalkOnline** [comment, marked as answer]
Die Position des Zoom Fensters kann in den Optionen (Zahnrad oben rechts) gemeinsam mit der Schablone umgestellt werden (links oder rechts am Rand). Ebenfalls ist es möglich das Zoom Fenster vom Rand abzudocken um es frei verschieben zu können.
![SemOnOptionen](https://user-images.githubusercontent.com/105418212/191233881-bc136c66-e2fc-43ad-a063-ed217aeec48a.PNG)
